package.json

```diff
@@ -52,7 +58,25 @@
     "run-sequence": "^0.3.6",
     "sinon": "^1.10.3",
     "through2": "^2.0.0",
-    "vinyl-sourcemaps-apply": "^0.1.4"
+    "vinyl-sourcemaps-apply": "^0.1.4",
+    "webpack": "^4.15.1",
+    "webpack-cli": "^3.0.8"
+  },
+  "dependencies": {
+    "ace-builds": "1.3.3",
+    "backbone": "^1.0.0",
+    "bootstrap": "^3.1.1",
+    "es5-shim": "^3.4.0",
+    "escodegen": "^1.10.0",
+    "escope": "^1.0.0",
+    "esprima": "^4.0.0",
+    "handlebars-loader": "^1.7.0",
```

Comment:
Do we still need the "handlebars" dependency if we're using "handlebars-loader"?

Reply:
Good Q. We import handlebars/runtime in SQL in order to register a helper dynamically, so we still use it. FOR NOW... BUM BUM BUM.